src/align_oc_msg_colons.cpp

```diff
@@ -124,10 +124,10 @@ void align_oc_msg_colon(chunk_t *so)
    {
       chunk_t chunk;
 
-      chunk.type        = CT_SPACE;
+      set_chunk_type(&chunk, CT_SPACE);                    // Issue #2567
```

Comment:
BTW, I'm... not sure what is the significance of this comment? #2567 isn't an issue, it is this PR. Did you mean to use a different number? I'm not sure how useful it is to annotate what PR changed code; we have `git blame` and such for that.

Reply:
Right. Thanks.